spec/requests/flows/authorization_code_spec.rb

```diff
@@ -28,8 +28,8 @@
       config_is_set(:token_secret_strategy, ::Doorkeeper::SecretStoring::Sha256Hash)
     end
 
-    scenario "Authorization Code Flow with hashing" do
-      @client.redirect_uri = Doorkeeper.configuration.native_redirect_uri
+    def authorize(redirect_url)
```

Comment:
Why do we removed `scenario "Authorization Code Flow with hashing` ?

Reply:
I didn't really remove the scenario per se, more added two variants and changed the name(s), testing `oob` and `oob:auto`. Since these scenarios happen within the feature 'Authorization Code Flow' and context 'with grant hashing enabled' I didn't think the scenario as named was saying much. The two new scenarios I guess would read 

1. Authorization Code Flow with grant hashing enabled using redirect_url `urn:ietf:wg:oauth:2.0:oob`.
2. Authorization Code Flow with grant hashing enabled using redirect_url `urn:ietf:wg:oauth:2.0:oob:auto`.

If expanded out.

Happy to change to whatever folks would prefer.